src/dune_rules/pkg_rules.ml

```diff
@@ -998,37 +1007,53 @@ module Fetch = struct
 
     let is_useful_to ~distribute:_ ~memoize = memoize
 
+    let encode_loc f (loc, x) =
+      Dune_lang.List
+        (* TODO use something better for locs here *)
```

Comment:
Do we have some common loc representation in Sexp? Like `(loc (file f.ml) (line 42))`?

Reply:
Not that I could find.